x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/unified_components/query_tab.cy.ts

```diff
@@ -17,22 +23,20 @@ import {
   TIMELINE_DETAILS_FLYOUT,
   USER_DETAILS_FLYOUT,
 } from '../../../../screens/unified_timeline';
-import {
-  ROW_ADD_NOTES_BUTTON,
-  ADD_NOTE_CONTAINER,
-  RESOLVER_GRAPH_CONTAINER,
-} from '../../../../screens/timeline';
-import { OPEN_ANALYZER_BTN } from '../../../../screens/alerts';
 import { GET_DISCOVER_DATA_GRID_CELL_HEADER } from '../../../../screens/discover';
 import { addFieldToTable, removeFieldFromTable } from '../../../../tasks/discover';
 import { login } from '../../../../tasks/login';
 import { visitWithTimeRange } from '../../../../tasks/navigation';
 import { openTimelineUsingToggle } from '../../../../tasks/security_main';
-import { createNewTimeline, executeTimelineSearch } from '../../../../tasks/timeline';
+import {
+  createNewTimeline,
+  executeTimelineSearch,
+  openTimelineInspectButton,
+} from '../../../../tasks/timeline';
 import { ALERTS_URL } from '../../../../urls/navigation';
+import { openTab } from '../../../../tasks/inspect';
 
-// FLAKY: https://github.com/elastic/kibana/issues/181882
-describe.skip(
+describe(
```

Comment:
🤦🏼‍♂️ Sorry for that. Let me fix it.

Reply:
Hey @angorayc , I have reverted to `EuiCodeBlock` since it had the support for virtualization. This will enable Fullscreen and copy to clipboard functions again.

Could you please check now.

Changes are [here](e849af5e504)